server/tests/test_smoke.py

```diff
@@ -263,12 +263,13 @@ async def test_working_on_v1_modules(client, caplog):
     with open(req_file, "r") as f:
         reqs = list(parse_requirements(f.readlines()))
 
+    str_reqs = [str(e) for e in reqs]
     venv = env.VirtualEnv(env_path)
     venv.use_virtual_env()
 
     assert Requirement.parse("inmanta-module-dummy>=3.0.8") in reqs
```

Comment:
We cannot just drop pkg_resources yet (until a new stable version has been released)

Reply:
I think I'm missing some context here, but doesn't the fact that the test fails means that people will encounter that same issue when opening a v1 module in vscode ?

Reply:
I'm not sure if this will break the VScode experience, so I'm asking for your review @wouterdb